test-common/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'java'
+}
+
+group 'com.synopsys.integration'
+version '6.4.0-SNAPSHOT'
```

Comment:
Are these needed here? Shouldn't the group and version be set by the main project, just like the other sub-projects?

Reply:
Yep. This was generated; I'm gonna clean it up right now.